.github/workflows/dapr-perf.yml

```diff
@@ -135,9 +135,9 @@ jobs:
       - name: Run Perf test actor_timer
         if: env.TEST_CLUSTER != ''
         env:
-          DAPR_PERF_QPS: 17
-          DAPR_PERF_CONNECTIONS: 10
-          DAPR_TEST_DURATION: 30m
+          DAPR_PERF_QPS: 1000
+          DAPR_PERF_CONNECTIONS: 16
+          DAPR_TEST_DURATION: 1m
```

Comment:
should this be 1 min or 10 minutes?

Reply:
This is 1 min on purpose. We want to have a big traffic spike to activate actors in the beginning and then wait for 10min (sleep inside the code) to collect metrics.